airbyte-integrations/connectors/source-audience-project/source_audience_project/schemas/campaigns.json

```diff
@@ -0,0 +1,168 @@
+{
+  "$schema":"http://json-schema.org/schema#",
+  "type":"object",
+  "additionalProperties": true,
+  "properties":{
+    "id":{
+      "type":"string"
```

Comment:
Make sure all schemas follow the reference https://docs.airbyte.com/connector-development/schema-reference